composer.json

```
@@ -34,6 +34,7 @@
     "require-dev": {
         "symfony/http-client": "^5.1",
         "symfony/browser-kit": "^5.1",
+        "symfony/polyfill-uuid": "1.19.0",
```

```suggestion
        "symfony/polyfill-uuid": "^1.19.0",
```

That way people using PHP 7+ won't be stuck with this version.